.github/workflows/e2e.yaml

```diff
@@ -43,6 +44,33 @@ jobs:
                 run: composer install --ansi
                 working-directory: ${{ matrix.directory }}
 
+            # run rector. check whether we expect a certain output, or alternatively that rector just processed everything without errors
             -
-                run: ./../../bin/rector process src --dry-run --ansi -a vendor/autoload.php
+                shell: php {0}
                 working-directory: ${{ matrix.directory }}
+                run: |
+                    <?php
+                    exec('./../../bin/rector process --dry-run --no-ansi --no-progress-bar -a vendor/autoload.php', $output, $exitCode);
+                    $output = trim(implode("\n", $output));
+
+                    $expectedDiff = 'expected-output.diff';
+                    if (!file_exists($expectedDiff)) {
+                        echo $output;
+                        exit($exitCode);
+                    }
+
+                    $matchedExpectedOutput = false;
+                    $expectedOutput = trim(file_get_contents($expectedDiff));
+                    if ($output === $expectedOutput) {
+                        $matchedExpectedOutput = true;
+                    }
+
+                    if ($matchedExpectedOutput === false) {
+                        echo "\nEXPECTED:\n";
+                        var_dump($expectedOutput);
+                        echo "\nACTUAL:\n";
+                        var_dump($output);
+                        exit(1);
+                    }
+
+                    exit(0);
```

Comment:
There should be no PHP in yaml/bash files. Could you extract this one to a test case file?

Reply:
@staabm with your latest patch, actually the command:

```bash
./../../bin/rector process --dry-run --no-ansi --no-progress-bar -a vendor/autoload.php
```

already OK:

```bash
➜  plain-views git:(start-end) ./../../bin/rector process --dry-run --no-ansi --no-progress-bar -a vendor/autoload.php

 [OK] Rector is done!                                                                                                   
```

so no need to check expected output.

Reply:
> so no need to check expected output.

hmm I guess you had adjusted the expected file or similar... I cannot reproduce on my end, that rector now is running like you described. it only happens when I change the view.php to a point, where it now longer reproduces the initial issue.

I went ahead and extracted a `e2eTestRunner.php` out of the github action workflow